### PR TITLE
license/ web page renders without 500 error

### LINKED ISSUE #424: license/ web page fails with 500 error
Accessing the license/ web page fails with a "500" error

### PR DESCRIPTION
**Pull request by lf32**
Closes #424

https://github.com/nexB/scancode.io/blob/16644a60c6dbe81fa078fc6941fbd8be817fe935/scancodeio/urls.py#L53

include function is updated

### CODE REVIEW

**Review by pombredanne — COMMENTED** [review]
Thanks! This is looking great. Can you add a test too?

**Comment by lf32** [comment]
wrote test cases and moved all urls to main urls.py

**Comment by tdruez** [comment]
@lf32 what's the reason to move the URL patterns outside their related module?

**Comment by lf32** [comment]
using licenses as namespace in includes() was throwing errors

**Comment by lf32** [comment]
if possible license urls can be seperated by placing in `license_urlpatterns` list 
and then `urlpatterns += license_urlpatterns` just like `auth_urlpatterns`
```
license_urlpatterns = [
    ...
]

urlpatterns = auth_urlpatterns + license_urlpatterns + [
   ...
]
```
would it be ok?

**Comment by tdruez** [comment]
@lf32 a better solution is to simply remove the "license_app" namespace during the registration at https://github.com/nexB/scancode.io/blob/main/scancodeio/urls.py#L53

Also, you new TestCase does not follow existing naming convention for tests:

Suggestions:
- get_licenses_test -> test_license_list_view
- get_license_details_test -> test_license_details_view

Lastly, the code is not properly formatted, make sure to run make valid before committing changes.

File: scancodeio/urls.py [hunk 1]
```diff
@@ -47,10 +47,11 @@
     path("accounts/profile/", AccountProfileView.as_view(), name="account_profile"),
 ]
 
+
 urlpatterns = auth_urlpatterns + [
     path("admin/", admin.site.urls),
     path("api/", include(api_router.urls)),
-    path("license/", include((licenses.urls, "license_app"))),
```
Comment (tdruez) [inline scancodeio/urls.py:53]: No reasons to rename the initial "license".

File: scanpipe/tests/test_licenses.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+# Visit https://github.com/nexB/scancode.io for support and download.
+
+from unittest import mock
+
+from django.test import TestCase
+from django.urls import reverse
+
+
+class LicensesTest(TestCase):
+    """docstring for LicensesTest"""
```
Comment (tdruez) [inline scanpipe/tests/test_licenses.py:30]: This docstring adds no value, please remove.

**Review by tdruez — CHANGES_REQUESTED** [review]
@lf32 Both tests are failing on the CI. Please fix so we can proceed with the merge ;)

File: scancodeio/urls.py (hunk 1 above)
Comment (lf32) [reply scancodeio/urls.py:53]: ok

**Comment by tdruez** [comment]
@lf32 Looking good! Thanks for taking care of this ⭐

**Comment by lf32** [comment]
@tdruez Thanks for merging this

**Pull request merged**